integration_tests/integration/part-b.spec.js

```diff
@@ -109,6 +109,56 @@ context('Part B', () => {
     })
   })
 
+  it('errors - uploads contain viruses', () => {
+    const recallId = '789'
+    const recalls = [
+      {
+        ...getEmptyRecallResponse,
+        recallId,
+        confirmedRecallType: 'STANDARD',
+        status: 'AWAITING_PART_B',
+      },
+    ]
+    cy.task('expectListRecalls', {
+      expectedResults: recalls,
+    })
+    cy.task('expectAddPartBRecord', {
+      statusCode: 400,
+      body: {
+        fileErrors: [
+          {
+            category: 'PART_B_RISK_REPORT',
+            fileName: 'part b.pdf',
+            error: 'VirusFoundException',
+          },
+          {
+            category: 'PART_B_EMAIL_FROM_PROBATION',
+            fileName: 'email.msg',
+            error: 'VirusFoundException',
+          },
+        ],
+      },
+    })
+
+    const partBRecord = getRecallResponse.partBRecords[0]
+    cy.visitRecallPage({ recallId, pageSuffix: 'part-b' })
+    cy.pageHeading().should('equal', 'Upload documents')
+    cy.uploadPDF({ field: 'partBFileName', file: 'test.pdf' })
+    cy.uploadPDF({ field: 'oasysFileName', file: 'test.pdf' })
```

Comment:
does it not matter that these file names dont match the filenames in the error response??

Reply:
Actually I can match them up as the OASys doesn't have an error; I've done that